chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -1,17 +1,66 @@
 {
 	"info": {
-		"_postman_id": "21c26d32-3ea4-4d99-ac3d-c4c557baa398",
 		"name": "developer-controller",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
+		{
+			"name": "set auth token",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "e86c22fc-2510-46ff-b7f6-7db4c2add64e",
+						"exec": [
+							"var authRes = JSON.parse(responseBody);\r",
+							"if(data.KeyName = \"token\")\r",
+							"{\r",
+							"pm.environment.set(\"authToken\", authRes.token);\r",
+							"}"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					},
+					{
+						"key": "API-KEY",
+						"value": "{{apiKey}}",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\"userName\":\"{{username}}\",\"password\":\"{{password}}\"}"
+				},
+				"url": {
+					"raw": "{{url}}/auth/authenticate",
+					"host": [
+						"{{url}}"
+					],
+					"path": [
+					    "rest",
+						"auth",
+						"authenticate"
+					]
+				}
+			},
```

Comment:
This whole section of getting auth should be part of pre-request script in the request where it is getting used( In PUT request)